### PR TITLE
Android10 updates part - 3

### PR DESCRIPTION
**Pull request by arminn**

### CODE REVIEW

**Comment by rshym** [comment]
1) "xenvm: Remove low_ram property" - tested, results in better ui.

2) Maybe "input: Remove old idc configuration" and "input: Add input port association file" should be swapped? I mean that maybe it's better to provide "input port association" and only after that remove .idc. Just in case if removal of .idc will left system not functional.

**Comment by rshym** [comment]
LGTM

**Review by rshym — APPROVED** [review]

**Comment by arminn** [comment]
> Just in case if removal of .idc will left system not functional.

It is not functional now, even with idc, since it needs some specific code add-ons to input flinger. So I suppose: 1 patch remove, 1 add new is ok.

File: device.mk [hunk 1]
```diff
@@ -251,6 +251,11 @@ PRODUCT_PACKAGES += \
 PRODUCT_PACKAGES += \
     Browser2 \
 
+# App for graphics testing
+PRODUCT_PACKAGES += \
+    glmark2
+
+
```
Comment (rshym) [inline device.mk]: Seems like this empty line is not needed.

**Comment by rshym** [comment]
Few comments regarding commit message of 5e2bc15 "xenvm: Do not copy configuration on to system partition".
- It's better to use 'vendor and oem' instead of 'vendor,oem', so it will be easier to identify that 'vendor and oem' is one logical thing, and 'system' - is other.
- There are two small misprints: use 'system partition' instead of 'sysytem partitions'.

File: device.mk (hunk 1 above)
Comment (arminn) [reply device.mk]: done

**Comment by arminn** [comment]
done - 5e2bc15 "xenvm: Do not copy configuration on to system partition".

**Comment by rshym** [comment]
Small misprint in commit message of 64363d "xenvm: Do not copy configuration on to system partition" - sysYtem.
With this small fix - looks good to me.

**Review by rshym — APPROVED** [review]

**Pull request merged**